pom.xml

```diff
@@ -2,18 +2,18 @@
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>net.hifis.cloud.marketplace</groupId>
-	<artifactId>hifis-marketplace</artifactId>
+	<groupId>de.helmholtz.marketplace</groupId>
```

Comment:
Is there any policy in Helmholtz about the management of namespaces? If yes - should we register this package name somewhere?

Reply:
I'm not aware of any namespace policy